xcc/job.py

```diff
@@ -93,23 +93,32 @@ class Job:
 
     @staticmethod
     def list(
-        connection: Connection, limit: int = 5, ids: Optional[Collection[str]] = None
+        connection: Connection,
+        limit: int = 5,
+        ids: Optional[Collection[str]] = None,
+        status: str = None,
     ) -> Sequence[Job]:
         """Returns jobs submitted to the Xanadu Cloud.
 
+
         Args:
             connection (Connection): connection to the Xanadu Cloud
             limit (int): maximum number of jobs to retrieve
             ids (Collection[str], optional): IDs of the jobs to retrieve; if at
                 least one ID is specified, ``limit`` will be set to the length
                 of the ID collection
+            status (str, optional): status parameter used to filter the jobs list.
+                If status is empty, return all the jobs
 
         Returns:
             Sequence[Job]: jobs which were submitted on the Xanadu Cloud by the
             user associated with the Xanadu Cloud connection
         """
         size = len(ids) if ids else limit
-        response = connection.request("GET", "/jobs", params={"size": size, "id": ids})
+
+        response = connection.request(
+            "GET", "/jobs", params={"size": size, "id": ids, "status": status}
+        )
```

Comment:
We can consolidate this code with a little refactoring:
```suggestion
        params = {"size": size, "id": ids, "status": status}
        response = connection.request("GET", "/jobs", params=params)
```

Reply:
It should not be, just double checked by printing the URL for when `status = None` and it's not included